tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -4432,3 +4432,158 @@ TEST_F(EnergyPlusFixture, SurfaceGeometry_CreateInternalMassSurfaces)
     EXPECT_EQ("TFLOORZONESINTMASS", DataSurfaces::IntMassObjects(3).Name);
     EXPECT_EQ("T SW APARTMENT TFLOORZONESINTMASS", SurfaceTmp(17).Name);
 }
+
+TEST_F(EnergyPlusFixture, SurfaceGeometry_CheckForReversedLayers)
+{
+    bool RevLayerDiffs;
+    bool ExpectResult;
+    Construct.allocate(6);
+    Material.allocate(7);
+    
+    // Case 1a: Constructs with regular materials are a reverse of each other--material layers match in reverse (should get a "false" answer)
+    Construct(1).TotLayers = 3;
+    Construct(1).LayerPoint(1) = 1;
+    Construct(1).LayerPoint(2) = 2;
+    Construct(1).LayerPoint(3) = 3;
+    Construct(2).TotLayers = 3;
+    Construct(2).LayerPoint(1) = 3;
+    Construct(2).LayerPoint(2) = 2;
+    Construct(2).LayerPoint(3) = 1;
+    RevLayerDiffs = true;
+    ExpectResult = false;
+    CheckForReversedLayers(RevLayerDiffs, 1, 2, 3);
+    EXPECT_EQ(RevLayerDiffs, ExpectResult);
```

Comment:
Simpler to use EXPECT_FALSE(RevLayerDiffs);

Reply:
My bad...I should have used the EXPECT_TRUE and EXPECT_FALSE functions rather than these.